Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -428,18 +433,17 @@ def prepare_ip_list_get_output(response: dict) -> str:
     hr_output = {
         "IP List HREF": response.get("href", ""),
         "Name": response.get("name", ""),
-        "Created At": arg_to_datetime(response.get("created_at", "")).strftime(HR_DATE_FORMAT),  # type: ignore
-        "Updated At": arg_to_datetime(response.get("updated_at", "")).strftime(HR_DATE_FORMAT),  # type: ignore
-        "FQDNS": response.get('fqdns', [])[0].get('fqdn') if response.get('fqdns') else []
+        "Created At": None if not response.get("created_at") else arg_to_datetime(  # type: ignore
+            response.get("created_at")).strftime(HR_DATE_FORMAT),
+        "Updated At": None if not response.get("updated_at") else arg_to_datetime(  # type: ignore
+            response.get("updated_at", "")).strftime(HR_DATE_FORMAT),
+        "FQDNS": response.get('fqdns')[0].get('fqdn') if response.get('fqdns') else []
```

Comment:
This needs to be fixed - why is the default value an empty array if the expected value is a string? We should also output all the FQDNs if the IP list has multiple:

```
"FQDNs": ", ".join([fqdn_rec["fqdn"] for fqdn_rec in response.get("fqdns", [])])
```

Reply:
I'll make the changes